src/EasyAbp.BookingService.Domain/EasyAbp/BookingService/AssetSchedules/DefaultAssetScheduleSelector.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+using Volo.Abp.DependencyInjection;
+
+namespace EasyAbp.BookingService.AssetSchedules;
+
+public class DefaultAssetScheduleSelector : IAssetScheduleSelector, ITransientDependency
+{
+    private readonly ILogger<DefaultAssetScheduleSelector> _logger;
+
+    public DefaultAssetScheduleSelector(ILogger<DefaultAssetScheduleSelector> logger)
+    {
+        _logger = logger;
+    }
+
+    public virtual Task<AssetSchedule> SelectAsync(List<AssetSchedule> assetSchedules,
+        DateTime date, TimeSpan startingTime, TimeSpan duration)
+    {
+        // TODO 部分满足是否属于该排期?
+        assetSchedules.RemoveAll(x => !x.ContainsTimeRange(date, startingTime, duration));
```

Comment:
Schedule1: 10:00-10:30
Schedule2: 10:30-11:00
Period1: 10:00-11:00

Period1 should match Schedule1 and Schedule2.